backend/app/api/v1/endpoints/overtime_records.py

```
@@ -0,0 +1,320 @@
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Unexpected error")
+
+@router.get(
+    "/{overtime_id}",
+    response_model=OvertimeRecordOut,
+    summary="Get overtime record by ID",
+    description="Retrieve a specific overtime record by its ID."
+)
+async def get_overtime_record_endpoint(
+    overtime_id: int,
+    request: Request,
+    current_user: Users = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)